### ISSUE TITLE
Finding % Accuracy during training time 

### ISSUE THREAD

**Issue by ShibaPrasad** [fizyr/keras-retinanet#825]
Hi,
I am currently training the retina-net model with my dataset (around 5000 images). I would like to modify the code:  So that i will be able to check % accuracy during the training time of each iteration. 

**Currently i am getting below message**
881/1000 [=========================>....] - ETA: 8:35 - loss: 2.7176 - regression_loss: 2.2216 - classification_loss: 0.4960./../../keras_retinanet/preprocessing/generator.py:165: UserWarning: Image with id 108 (shape (74, 74, 3)) contains the following invalid boxes: [[45. 13. 75. 53.]

But I would like to display the % validation loss on each iteration along with above 2 losses. My main intention is to know - at what point I need to stop training  i.e  - let say 90% accuracy level. 
Something similar to (https://github.com/keras-team/keras/issues/2548)

Can I do that? If Yes then which part of the code i need to change and any thing i need to update?

**Comment by hgaiser** [comment]
The validation loss isn't computed, but shouldn't be too hard to add. I opted not to add it, but have it compute the validation mAP instead, since that is more indicative of the quality of the network. You can check the `--help` to see how to do validation accuracy computations.

**Comment by ShibaPrasad** [comment]
Thank you.
I used "./train.py --steps=100 --epochs=50 csv path-to/data.csv path-to/Class_mapping.csv --val-annotations path-tovalidation/Singlexml_data.csv" for my validation and also getting the mAP.

Would you please help me - understanding technically - how should I indicate that my job working fine with the mAP values (increasing or decreasing values). What should be ideal.
